project/AppDependencies.scala

```diff
@@ -15,9 +15,7 @@ object AppDependencies {
 
   val test = Seq(
     "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % "5.14.0"              % Test,
-    "com.vladsch.flexmark"    %  "flexmark-profile-pegdown" % "0.62.2"              % Test,
-    "org.scalacheck"          %% "scalacheck"               % "1.15.0"              % Test,
```

Comment:
scalatestplus-scalacheck has a transitive dependency on the appropriate version of scalacheck.